src/kart-analytics.py

```diff
@@ -1,46 +1,49 @@
 import pandas as pd
 import matplotlib.pyplot as plt
 
-# reda csv file
-df = pd.read_csv('kart-data.csv', na_values=['', 'NA', 'N/A', 'NaN'])
+# Read csv file
+df = pd.read_csv(r'src/kart-data.csv', na_values=['', 'NA', 'N/A', 'NaN'])
```

Comment:
This line can't work, the csv file is in the same directory as the python source file. (I also tested)
So I'm gonna commit on this change

Reply:
Yeah for some reason my local repo doesnt work without the src at the front of the path. its odd because they are in the same folder any way thanks for merging my request this is first contribution to open source and I really enjoy it!

Reply:
No problem, glad to hear!
 Feel free to give a look on my profile if you would like to contribute on other project(or this one again)